app/views/components/p2.scala.html

```diff
@@ -0,0 +1,21 @@
+@*
```

Comment:
Not too sure if I'm missing the reasoning behind this but looks like it matches the p.scala.html component code.

Reply:
Nope - you're correct. 
I was thinking of introducing a new element for one of the divs but it got too complicated. Forgot to remove it!